README.md

```diff
@@ -129,23 +180,24 @@ If you like this project please let me know
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-My mother. Let's play to the end together ❤️ with 🧠
-<br/>
+We would like to thank our family, all the partners and Microverse platform.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## ❓ FAQ <a name="faq"></a>
 
-- **Is there any restriction to adapting the template to other porpouse??**
+- **is there any restriction to put my name instead of the original author's name?**
 
-  - No problem.
+  - You can delete my name and put yours without any problem.
+
+- **Can I change the original colors or assign a new color palette?**
+
+  - Yeah, no problem
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](<a href="https://github.com/chelobotix/Leaderboard-Chelobotix/blob/api-request/LICENSE">License</a>) licensed.
-
-<a href="https://github.com/chelobotix/Leaderboard-Chelobotix/blob/api-request/LICENSE">License</a>
+This project is [MIT]([./LICENSE](https://github.com/chelobotix/Retro-Watchers-Capstone/blob/develop/LICENSE)) licensed.
```

Comment:
- [ ] The link you used here for the `LICENCE` file is not working. Kindly fix this issue to make your project more professional.

![image](https://user-images.githubusercontent.com/60083437/219792433-f5c366c7-e448-4ecf-9715-0158d649b379.png)